automation/prepare-release.py

```diff
@@ -40,6 +40,16 @@
 
 ensure_working_tree_clean()
 
+step_msg(f"Calculating remote owner {remote}")
+result = run_cmd_checked(["git", "remote", "get-url", remote], stdout=subprocess.PIPE)
+repo_url = result.stdout.decode('utf8').strip()
+print(repo_url)
+match = re.match(r"git@github.com:(.*?)/application-services.git", repo_url)
```

Comment:
Do we want to also support `https` here?  Not sure if it even works now, so ignore this if it already doesn't

On that note, if we decide that we only support `ssh` remote url's we might wanna add a quick note about it in the `cutting a release doc` - again if that already exists, ignore this 😬